cmd/fence-provision-manager/ansible/roles/power/tasks/deprovision.yml

```diff
@@ -0,0 +1,6 @@
+##############################################################################
+## Power off an machine with IPMI
+##############################################################################
+
+- name: Power off machine
+  command: "/sbin/fence_{{ agent_type }} -P --ip {{ server }} --ipport {{ ipport }} --username {{ username }} --password {{ password }} -o off"
```

Comment:
You're assuming that all agents need those parameters, and only those parameters - which is unfortunately not the case.  Worse, they like to error out when they are given an option that don't need.

Fine for now since we only have access to IPMI based hardware, but we'll need to address it later

Reply:
Agree, but it is the problem with ansible, to pass a parameter via `extra-vars` you must specify this parameter under playbook via `{{ param }}`, so you will need to specify all fence_agent parameters and provide files with defaults(in case if a user did not specify this parameter under `Config` or `DynamicConfig`), I thought to add something like `{{ extra_args }}` and pass as
`--extra-vars "extra_args=\"-P --ipport 6230\""` but to be honest it looks ugly.
If we will continue with ansible we will need to find some solution to it.